Sources/SwiftParser/Types.swift

```diff
@@ -844,42 +857,9 @@ extension Parser.Lookahead {
     preconditionFailure("Should return from inside the loop")
   }
 
-  mutating func canParseOldStyleProtocolComposition() -> Bool {
-    self.eat(.protocolKeyword)
-
-    // Check for the starting '<'.
-    guard self.currentToken.starts(with: "<") else {
-      return false
-    }
-
-    self.consumePrefix("<", as: .leftAngle)
-
-    // Check for empty protocol composition.
-    if self.currentToken.starts(with: ">") {
-      self.consumePrefix(">", as: .rightAngle)
-      return true
-    }
-
-    // Parse the type-composition-list.
-    var loopProgress = LoopProgressCondition()
-    repeat {
-      guard self.canParseTypeIdentifier() else {
-        return false;
-      }
-    } while self.consume(if: .comma) != nil && loopProgress.evaluate(currentToken)
-
-    // Check for the terminating '>'.
-    guard self.currentToken.starts(with: ">") else {
-      return false
-    }
-    self.consumePrefix(">", as: .rightAngle)
-
-    return true
-  }
```

Comment:
@bnbarham Was this removed on purpose?

Reply:
@AnthonyLatsis Yes, in `SwiftParser` we decided not to support old style protocol composition (there's no `parseOldStyleProtocolComposition()`.) So keeping `canParseOldStyleProtocolComposition()` doesn't make sense.